core/server/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -277,8 +282,8 @@ public CreatePathResult createPath(AlluxioURI path, CreatePathOptions options)
             .append(traversalResult.getNonexistentPathIndex()).append("(")
             .append(parentPath[traversalResult.getNonexistentPathIndex()])
             .append(") does not exist").toString();
-        LOG.info("InvalidPathException: {}", msg);
-        throw new InvalidPathException(msg);
+        LOG.info("FileDoesNotExistException: {}", msg);
```

Comment:
I'd lean more towards debug since it's an expected exception and we throw it. Do we have a convention for when to log exceptions?